v2/emailpassword/advanced-customizations/apis-override/usage.mdx

```diff
@@ -75,4 +77,82 @@ SuperTokens.init({
+In this example we will be overriding the signIn API.
+:::
+
+### Step 1: Copy the API code from the SDK.
+
+We first need to find the implementation for signInPOST, this can be found [here](https://github.com/supertokens/supertokens-node/blob/master/lib/ts/recipe/emailpassword/api/implementation.ts#L95).
+
+### Step 2: Add the API code in the override config.
+Now in the override config instead of calling the original implementation like we did in [Method 1](#method-1-use-the-override-config) we can paste the signInPOST code we copied earlier and modify it.
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+```jsx
+SuperTokens.init({
+    appInfo: {...},
+    supertokens: {...},
+    recipeList: [
+        EmailPassword.init({
+            // highlight-start
+            override: {
+                apis: (originalImplementation) => {
+                    return {
+                        ...originalImplementation,
+
+                        signInPOST: async (input) => {
+
+                            // we add the default implementation code we copied from the SDK and add custom logic.
+                            let newSignInPost = async ({ formFields, options }) => {
+                                
+                                let email = formFields.filter((f) => f.id === "email")[0].value;
+                                let password = formFields.filter((f) => f.id === "password")[0].value;
+
+                                let response = await options.recipeImplementation.signIn({ email, password });
+                                if (response.status === "WRONG_CREDENTIALS_ERROR") {
+                                    return response;
+                                }
+                                let user = response.user;
+
+                                let jwtPayloadPromise = options.config.sessionFeature.setJwtPayload(user, formFields, "signin");
+                                let sessionDataPromise = options.config.sessionFeature.setSessionData(user, formFields, "signin");
+
+                                let jwtPayload = await jwtPayloadPromise;
+                                let sessionData = await sessionDataPromise;
+
+                                await Session.createNewSession(options.res, user.id, jwtPayload, sessionData);
+                                return {
+                                    status: "OK",
+                                    user,
+                                };
+                            }
+
+                            let response = await newSignInPost(input);
+                            if (response.status === "OK") {
+                                let { id, email } = response.user;
+
+                            }
+
+                            return response;
+                        },
+                        
+                    }
+                },
+            }
+            // highlight-end
+        })
+    ]
+});
+```
+</TabItem>
+</BackendSDKTabs>
```

Comment:
Write after:
After pasting, you can freely modify any step in the API. For example, if you want to modify the API so that a new session is created only if a user is not logged into another device, you can add the logic for that right before calling the `await Session.createNewSession` function.